src/vdaf.rs

```diff
@@ -460,161 +460,171 @@ impl<F: FieldElement> Encode for AggregateShare<F> {
     }
 }
 
```

Comment:
Reviewer note: Recommend hiding whitespace. All we've done here is move run_vdaf and run_vdaf_prepare into a module.